src/interface/sparse_pack_base.hpp

```diff
@@ -64,6 +64,8 @@ class SparsePackBase {
   using block_props_h_t = typename block_props_t::HostMirror;
   using coords_t = ParArray1DRaw<ParArray0D<Coordinates_t>>;
 
+  static constexpr int bnd_flag = -2000;
```

Comment:
Sparse pack stores the refinement level of each blocks neighbors in every direction, which Ben R. added for swarms. Now, if the level is set to this value (which I changed to `physical_bnd_flag`), it denotes that a given side of the block abuts a physical boundary. This is necessary for the boundary aware prolongation routines. This number is negative and large enough it will never collide with an actual refinement level.